recipes/diligent-core/all/conanfile.py

```diff
@@ -78,6 +72,28 @@ def package_id(self):
             else:
                 self.info.settings.compiler.runtime = "MT/MTd"
 
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["DILIGENT_BUILD_SAMPLES"] = False
+        tc.variables["DILIGENT_NO_FORMAT_VALIDATION"] = True
+        tc.variables["DILIGENT_BUILD_TESTS"] = False
+        tc.variables["DILIGENT_NO_DXC"] = True
+        tc.variables["DILIGENT_NO_GLSLANG"] = not self.options.with_glslang
+        tc.variables["SPIRV_CROSS_NAMESPACE_OVERRIDE"] = self.options["spirv-cross"].namespace
+        tc.variables["BUILD_SHARED_LIBS"] = False
+        tc.variables["DILIGENT_CLANG_COMPILE_OPTIONS"] = ""
+        tc.variables["DILIGENT_MSVC_COMPILE_OPTIONS"] = ""
+        tc.variables["ENABLE_RTTI"] = True
+        tc.variables["ENABLE_EXCEPTIONS"] = True
+        tc.variables[self._diligent_platform()] = True
+        tc.generate()
+
+        deps = CMakeDeps(self)
+        deps.generate()
+
+    def layout(self):
+        cmake_layout(self)
```

Comment:
```suggestion
        cmake_layout(self, src_folder="src")
```

Reply:
Please, try your changes first.
This breaks the build.

Reply:
@jwillikers This recipe is a bit different, it uses a cmake wrapper, so using src is not required, because we still use source_subfolder. We could add src/, but it would need to use src/source_subfolder, or update the recipe. Another trick would be copying the wrapper to src/ and renaming the wrapper and configure CMake to call the wrapper (I didn't test it).